docs/ARCHITECTURE.md

```diff
@@ -1,70 +1,33 @@
-## Architectural Summary
+# Technical Architecture
 
-- REST API
-- Frontend
-- Database
-- Membership Authentication/Authorization
-  - KeyCloak
-  - KeyCloak DB
-- GIS Map interface
-  - Leaflet
-- DevOps
-  - Containerized Docker solution
-  - Build/Deploy scripts
+The PIMS solution follows a standardized approach which keeps each component independent and within its own container. This allows for scalability that can be applied dynamically when required. As the solution is developed the architecture will reflect this change by becoming more complex and including additional components (i.e. caching, load balancing, services).
 
-## Architectural Design
-All components of the solution are configured for Docker containers to run independently and therefore scallable.
+![solution architecture](./images/arch-01.png)
 
-### APP
-The APP frontend will be a React application that will provide the user interface.  It will provide authentication/authorization with KeyCloak (Using Open ID Connect).  To communicate with the API it will proxy requests that begin with `/api` to the configured API host.
+### Membership
 
-It will be written in React and have a Node backend.
+PIMS access requires a validated account to access. A user with one of the following account; IDIR, BCeID, GitHub or a custom.
```

Comment:
one of the following account(s)

also are we planning on allowing github or custom accounts access to PIMS?

Reply:
I've changed the text to say optional.